src/index.js

```diff
@@ -1,4 +1,8 @@
 'use strict';
+
+// Enable core-js polyfills
```

Comment:
I still don't know what this is for. Like what functionality are we using that makes us need this, what does it provide? Why was it working without this before?